docs/reference/query-dsl/rank-feature-query.asciidoc

```diff
@@ -203,23 +291,21 @@ GET test/_search
 }
 --------------------------------------------------
 // CONSOLE
-// TEST[continued]
-
-[float]
-==== Sigmoid
 
-This function is an extension of `saturation` which adds a configurable
+[[rank-feature-query-sigmoid]]
+===== Sigmoid
+The `sigmoid` function is an extension of `saturation` which adds a configurable
```

Comment:
Same thought here about the interval notation.

Reply:
Re-added with 890b6b3.